src/sentry/performance_issues/detectors/large_payload_detector.py

```diff
@@ -51,6 +78,16 @@ def visit_span(self, span: Span) -> None:
             encoded_body_size = int(encoded_body_size)
 
         if encoded_body_size > payload_size_threshold:
+            # Check if this is a file download that should be excluded - can be moved to `is_span_eligible` once the flag is removed
```

Comment:
filters here instead of `is_span_eligible` so we can get a better sense of the impact. if we filtered there, we wouldn't know if that span would match our criteria of detecting a problem